sidecar2cdxj.py

```diff
@@ -16,27 +15,23 @@ def create_cdxj_path(sidecar_file, archive_dir):
     return os.path.join(archive_dir, cdxj_file)
 
 
-def convert_payload_to_dict(record):
+def convert_payload_to_json(record):
     """Parse a record's payload, put the fields into a dictionary and return the dictionary."""
```

Comment:
We should update this docstring to reflect returning the JSON string...maybe "and return it as serialized JSON" or "and return it as a JSON string" or something like that.